app/assets/stylesheets/govuk_publishing_components/components/_action-link.scss

```diff
@@ -92,6 +92,20 @@
   }
 }
 
+.gem-c-action-link--white-arrow {
+  &:before {
+    content: "";
+    display: table-cell;
```

Comment:
You don't need to define `content` or `display` here - these are already set on the default `:before` element on this component - https://github.com/alphagov/govuk_publishing_components/blob/d65aeef6119a6a36938b437da7741d079defdae6/app/assets/stylesheets/govuk_publishing_components/components/_action-link.scss#L6